charts/node-red/templates/sidecar-cm.yaml

```diff
@@ -3,9 +3,11 @@ apiVersion: v1
 data:
   flow_refresh.py: | {{ range $.Files.Lines "scripts/flow_refresh.py" }}
     {{ . }}{{ end }}
+  extra-node-modules.json: |-
```

Comment:
Can you warp this around in an `if` statement please. In case, the user does not want extra modules.

Reply:
Thank you for your review. If statement `is` now included for the `extra-node-modules `!